config/locales/activerecord.yml

```diff
@@ -1,4 +1,18 @@
 en:
+  cookies_preferences_errors: &cookies_preferences_errors
+    cookies_consent:
+      inclusion: Please select an option
+  hiring_staff_user_preference_errors:
+    managed_organisations:
+      blank: Select which schools' job listings you would like to view
+  job_alert_feedback: &job_alert_feedback_errors
+    comment:
+      blank: You have not submitted any further feedback.
+      too_long: Feedback must not be more than 1,200 characters
```

Comment:
Is this @valentine-carter approved?

Reply:
`You have not submitted any further feedback.` is from Valentine.
`Feedback must not be more than 1,200 characters` is copied from the other errors. However, it will never be displayed unless we remove the field-level validation of the character limit.

Reply:
Ok cool, I was wondering if 'Feedback must be less than' was clearer, but it doesn't really matter